jwst/master_background/master_background_step.py

```diff
@@ -140,6 +146,12 @@ def process(self, input):
                         exptime_key='exposure_time',
                     )
 
+                    # If requested, apply a moving-median boxcar filter to the master background spectrum
+                    if (self.median_kernel > 1):
+                        self.log.info('Applying moving-median boxcar of width %i',self.median_kernel)
+                        master_background.spec[0].spec_table['surf_bright'] = medfilt(
```

Comment:
medfilt throws an error for even kernel sizes; should we add a check on median_kernel to see if the user supplied an even number, modify it to be a reasonable odd value (e.g., either round up or down to the nearest odd value), and warn users that their input was even and therefore rounded?  

I think we probably should have such a check to avoid running into an error while processing and it might make sense to round down to the nearest odd integer.  Does that sound reasonable?  If so I can make the change when I merge this into my branch.

Reply:
Sounds good to me.